moto/s3/models.py

```diff
@@ -937,6 +937,7 @@ def __init__(self, name: str, account_id: str, region_name: str):
         self.default_lock_days: Optional[int] = 0
         self.default_lock_years: Optional[int] = 0
         self.ownership_rule: Optional[Dict[str, Any]] = None
+        s3_backends.bucket_owners[name] = account_id
```

Comment:
This could be a module level variable but I feel like it belongs to the backend.